### ISSUE TITLE
Multiple choice options are always applies, even if not checked (Switch mode)

### ISSUE THREAD

**Issue by t3nk3y** [NiceneNerd/BCML#190]
Mods that have multiple choice options are are applying all options, whether or not those options are selected.  This obviously completely defeats the purpose of the feature.

I can provide an example BCML, info.json, and the source folders if you want.  But the mod I'm working on is a re-package of Zelda's Ballad, which is pretty big, so I don't want to just slap it up on a github issue.  Let me know if you want to work on addressing the issue together.  If you want, I'm available to chat on Discord, and am a very experienced developer, so I can provide full assistance at issue resolution.

**Comment by NiceneNerd** [comment]
Well, this was a silly problem. Fixed here: https://github.com/NiceneNerd/BCML/commit/03b124ead4d31131e6a5dbbec897e3f9784397c1#diff-15a439d49e846ef15a43c4f8fa56f59272b39187d4504f1533dd72aaf74b3df4.

On install, BCML checks to see if there are any options to use, and if so it deletes all unused options from the mod. Problem: the default "truthyness" of the options is negative whether it is empty (no options selected) or null (no options available). Thus I had to change it to explicitly check against a `None` value. The fix will be in the next release.

**Comment by t3nk3y** [comment]
Thanks, I confirmed that it fixed the issue in my local environment.
2 questions:

1.) Just to confirm, this meant that(before the fix) it would work when at least 1 option was selected, but when none where selected, it would install all of them.

2.) Any plans on a release timeline?  I want to hold off on releasing several mods till this is fixed, as I don't want to confuse a slew of users who run in to the same problem, as most of these mods will be using the options feature of BCML.  I also don't want to leave a bad impression of BCML for users who are using BCML mods with options for the first time.  I wouldn't be as cautious if these were brand new mods I built.  But really, they are repacks and conversions of Wii U editions for the Switch, for some very popular mods, like Zelda's Ballad, Girly Animations, Dialog and Custscene Audio Adjustments, Linkle Mod, Linkle Armor GUI Icons.

I'll be providing some of these to original maintainers, in other cases the mod author asked me to put up my own copy of the mod, but they plan to link from the Wii U to the switch.

I'm just wondering if it makes sense for me to hold off for a new release of BCML, or if I should re-pack these as separate BCML's and use the option feature later.
